plugin/src/main/groovy/com/novoda/staticanalysis/internal/detekt/DetektConfigurator.groovy

```diff
@@ -13,6 +13,7 @@ class DetektConfigurator implements Configurator {
     private static final String DETEKT_PLUGIN = 'io.gitlab.arturbosch.detekt'
     private static final String DETEKT_NOT_APPLIED = 'The Detekt plugin is configured but not applied. Please apply the plugin in your build script.\nFor more information see https://github.com/arturbosch/detekt.'
     private static final String OUTPUT_NOT_DEFINED = 'Output not defined! To analyze the results, `output` needs to be defined in Detekt profile.'
+    private static final String DETEKT_CONFIGURATION_ERROR = 'A problem occurred while configuring Detekt. Please make sure to use a compatible version.'
```

Comment:
Would be nice to write what versions are compatible

Reply:
Yes, very good point. I was thinking about it but I couldn't think about a way to do it in a maintainable way. It is at least compatible up to `RC8`. 

Considering this is shown only in case of a future incompatibility, it should be fine.